app/build.gradle

```diff
@@ -98,7 +98,8 @@ android {
         resources {
             // remove two files which belong to jsoup
             // no idea how they ended up in the META-INF dir...
```

Comment:
Please also add a comment adding that you are removing the COPYRIGHT file of RxJava here.

Reply:
Gotcha, I thought I did but I guess I forgot to when committing.